README.md

```diff
@@ -30,9 +133,9 @@ Several examples for using the bindigs are available in https://github.com/robot
 Then, properly accessing bindings to iDynTree can require some additional steps.
 #### Python
 You should add to the `PYTHONPATH` enviromental variable the install path of the `iDynTree.py` file.
-For a typical installation of the `codyco-superbuild`, this will require adding to the `.bashrc` a line similar to this:
+For a typical installation of the `robotology-superbuild`, this will require adding to the `.bashrc` a line similar to this:
 ~~~
-export PYTHONPATH=$PYTHONPATH:~/src/codyco-superbuild/build/install/lib/python2.7/dist-packages/
+export PYTHONPATH=$PYTHONPATH:~/src/robotology-superbuild/build/install/lib/python2.7/dist-packages/
```

Comment:
Here it would be better using `<ROBOTOLOGY_REPOSITORY>` or similar to avoid confusion. `~/src` is currently used.

Reply:
In the Matlab section the `{CMAKE_INSTALL_PREFIX}/mex` is referenced. I would explain the setup of these folders in a similar way to avoid confusion.

Reply:
Octave as well